packages/account/src/Components/poi/idv-document-submit/idv-document-submit.tsx

```diff
@@ -192,7 +192,7 @@ const IdvDocumentSubmit = observer(({ handleBack, handleViewComplete, selected_c
                             editable_fields={status?.is_confirmed ? [] : changeable_fields}
                         />
                     </section>
-                    <FormFooter className='proof-of-identity__footer'>
+                    <FormFooter className='proof-of-identity__footer account-form__footer--reset'>
```

Comment:
This change is to make footer section relative as the issue is happening on smaller height displays coz the footer had position: absolute so it was overlapping with the upper section